charts/kafka/experiments.yaml

```diff
@@ -51,10 +51,10 @@ spec:
       value: 'statefulset'
 
     - name: KAFKA_LIVENESS_STREAM
-      value: 'enabled'
+      value: 'enable'
 
     - name: KAFKA_LIVENESS_IMAGE
-      value: 'litmuschaos/kafka-client:ci'
+      value: 'litmuschaos/kafka-client:latest'
```

Comment:
PTAL @uditgaurav @ispeakc0de Are we good with the `latest` tag?

Reply:
yes